docs/explore/introduction/contributing.md

```diff
@@ -14,80 +14,145 @@ tags:
   cover to cover. It is a light read that touches on all subjects that you will
   need to know about superficially.
 
-- To get a general grasp of the project, read the
-  <a href="/api3-whitepaper-v1.0.3.pdf" target="_api3-whitepaper">API3
-  Whitepaper<ExternalLinkImage/></a> and our
-  [Blog Posts on Medium<ExternalLinkImage/>](https://medium.com/api3).
+## Non-technical contributors
 
-- Read the entirety of these docs. Take notes and cross-reference, but do not
-  expect to understand everything. Feel free to ask questions.
+Marketers, creatives and community builders can contribute to DAO subteams with
+ad-hoc projects or to support areas that are lacking focus. The challenge here
+is accurately scoping, forming a brief proposed solution method, and then
+delivering in alignment with other involved contributors.
+
+Contirbutions may look like:
 
-- Run and understand the Airnode starter tutorial
+- blog posts, community resources
+- ambassador activity, setting of events
+- art or branded merch
 
-- (Optional, but ideal) Integrate an API to an Airnode without using any guide,
-  deploy the Airnode, and build a dApp that uses the Airnode. Better yet, join
-  one of our hackathons.
+# How do I access funds to support my efforts? 
 
-- (For the curious) Explore all the repos hosted on
-  [GitHub<ExternalLinkImage/>](https://github.com/api3dao), go through the
-  issues, PRs, etc.
+Prospective contributors should provide a clear scope of work with (1) detailed
+deliverables and estimations of completion time (2) any applicable coordination
+with other contributors necessary to accomplish the deliverables and (3) the
+amount of API3 tokens requested.
 
-This page will be updated as more materials are produced. Please feel free to
-create
-[GitHub issues<ExternalLinkImage/>](https://github.com/api3dao/vitepress-docs/issues)
-to make suggestions.
+We appreciate the individual nature of contributing and these frameworks are
+flexible, yet linking these activities to the challenges outlined in team
+proposals will support our hollisitic objectives and be more likely supported.
 
-## GitHub
+## Ecosystem fast bounties & retropective grants
 
-Visit the [API3 GitHub<externalLinkImage/>](https://github.com/api3dao) page and
-find everything about.. everything. Explore the many repositories to gain
-in-depth knowledge about API3.
+The API3 Ecosystem team has a dedicated pool of funds to support micro-grants from community contributors. This can be retrospective or pre-planned, but has been created to make rewarding contribution frictionless and simple. 
 
-- the whitepaper
-- source code
-- documentation
-- usage examples
+## This can be submitted in a variety of way
 
-Please feel free to contribute to any API3 GitHub repositories.
+- Join the API3 [community<ExternalLinkImage/>](https://discord.gg/qnRrcfnm5W)
+and submit within the ecosystem contributor request channel. 
+- Submit an
+overview onto the [API3 forums](https://forum.api3.org/). 
```

Comment:
I would suggest using "sentiment check" instead of overview, and/or linking to https://docs.api3.org/guides/dao-members/proposals.html#proposal-creation